Dockerfile

```diff
@@ -14,12 +14,9 @@ RUN composer install --ignore-platform-reqs --optimize-autoloader \
 
 FROM node:16.14.2-alpine3.15 as node
 
-WORKDIR /usr/local/src/
+COPY app/console /usr/local/src/console
 
-COPY package-lock.json /usr/local/src/
-COPY package.json /usr/local/src/
-COPY gulpfile.js /usr/local/src/
-COPY public /usr/local/src/public
+WORKDIR /usr/local/src/console
 
 RUN npm ci
```

Comment:
To make better of the image cache, shouldn't we only copy the package.json and package-lock.json first, run `npm ci` and then copy the rest of the files and run `npm run build`?